locales/ara.json

```diff
@@ -97,7 +97,7 @@
     "offlineAuthenticationDisabled": "تفعيل لتسجيل الدخول عبر الإنترنت",
     "inProgress": "في تَقَدم",
     "credentialActivated": "مفعل",
-    "profileAuthenticated": "رمز الاستجابة السريعة تسجيل الدخول",
+    "profileAuthenticated": "رمز الاستجابة السريعة تسجيل الدخول"
```

Comment:
confirm this with latest develop and revert back if it doesn't match

Reply:
This was a JSON validation error, this had caused a lot of worry and confusion to us earlier.